### DISCUSSION TITLE
혹시 GetComponent(string) 를 사용할 수 있습니까?

### DISCUSSION THREAD

**Discussion by peterkangdev** [naverz/zepeto-studio-kor#1235] [5. Q&A]
월드 1.6.0 입니다.
GetComponent(string) 를 사용해 봤는데, 에러가 나서 문의 드립니다. 
사용가능 한가요? 아님 제가 잘못 사용하고 있는지??

**Comment by zlmrzlo** [comment]
GetComponent는 주로 인스팩터에 있는 컴포넌트에 접근하기 위해서 사용하는 것이라 string을 가지고 오는 것은 못 봤습니다.
그리고 GetComponent<string>() 이런 식으로 <> 사이에 변수 타입을 작성하셔서 사용하면 됩니다.

**Comment by Cha-SeongHun** [comment]
저도 ```GetComponent(string)``` 함수 사용했을 때 동작하지 않네요
제네릭 함수 ```GetComponent<T>()``` 이외에 ```string```을 파라미터로 넘기는 함수도 상황에 따른 수요가 있어서 고쳐져야 하는 이슈인 것 같아요
